src/components/include/protocol_handler/session_observer.h

```diff
@@ -103,8 +103,19 @@ class SessionObserver {
       const bool is_protected,
       const BsonObject* params) = 0;
 
-  // DEPRECATED
-  virtual uint32_t OnSessionEndedCallback(
+  /**
+   * \brief Callback function used by ProtocolHandler
+   * when Mobile Application initiates session ending.
+   * \param connection_handle Connection identifier within which session exists
+   * \param sessionId Identifier of the session to be ended
+   * \param hashCode Hash used only in second version of SmartDeviceLink
+   * protocol.
+   * If not equal to hash assigned to session on start then operation fails.
+   * \param service_type Type of service
+   * \return uint32_t 0 if operation fails, session key otherwise
+   * \deprecated
```

Comment:
@jacobkeeler do we have some unified style of Doxygen documentation? 
I mean marking keywords with backslash `\brief` or with *at* symbol `@brief`?
For example, [here](https://github.com/smartdevicelink/sdl_core/blob/a8c3e2ac28d758b30a2b933918635ef33f2d4cd3/src/components/application_manager/include/application_manager/application.h#L161-L163) we use *at* symbol.
And I like marking with *at* symbol

Reply:
I agree, and I suppose I could change it for this specific deprecated function. However, the rest of this file uses the `\` symbol instead of `@`. I was simply attempting to not break the consistency of this format within the file. It may be worth creating a PR changing all of the files to use this `@` symbol, but it is a minor enough detail that I don't think it should be a priority.